stdeb/util.py

```diff
@@ -1404,18 +1404,35 @@ def build_dsc(debinfo,
                          'compatible with older versions of debhelper.'%(
                     DH_MIN_VERS,))
 
-        python_defaults_version_str = get_version_str('python-all')
-        if len(python_defaults_version_str)==0:
-            log.warn('This version of stdeb requires python-all >= %s, '
-                     'but you do not have this package installed. '
-                     'Could not check compatibility.'%PYTHON_ALL_MIN_VERS)
-        else:
-            if not dpkg_compare_versions(
-                python_defaults_version_str, 'ge', PYTHON_ALL_MIN_VERS):
-                log.warn('This version of stdeb requires python-all >= %s. '
-                         'Use stdeb 0.6.0 or older to generate source packages '
-                         'that use python-support.'%(
-                    PYTHON_ALL_MIN_VERS,))
+        if debinfo.with_python2:
+            python_defaults_version_str = get_version_str('python-all')
+
+            if len(python_defaults_version_str)==0:
+                log.warn('This version of stdeb requires python-all >= %s, '
+                         'but you do not have this package installed. '
+                         'Could not check compatibility.'%PYTHON_ALL_MIN_VERS)
+            else:
+                if not dpkg_compare_versions(
+                    python_defaults_version_str, 'ge', PYTHON_ALL_MIN_VERS):
+                    log.warn('This version of stdeb requires python-all >= %s. '
+                             'Use stdeb 0.6.0 or older to generate source packages '
+                             'that use python-support.'%(
+                             PYTHON_ALL_MIN_VERS,))
+
+        if debinfo.with_python3:
+            python3_defaults_version_str = get_version_str('python3-all')
+
+            if len(python3_defaults_version_str)==0:
+                log.warn('This version of stdeb requires python3-all >= %s, '
+                         'but you do not have this package installed. '
+                         'Could not check compatibility.'%PYTHON_ALL_MIN_VERS)
+            else:
+                if not dpkg_compare_versions(
+                    python3_defaults_version_str, 'ge', PYTHON_ALL_MIN_VERS):
+                    log.warn('This version of stdeb requires python3-all >= %s. '
+                             'Use stdeb 0.6.0 or older to generate source packages '
+                             'that use python-support.'%(
+                             PYTHON_ALL_MIN_VERS,))
```

Comment:
The minimum version `PYTHON_ALL_MIN_VERS` only applies to Python 2. See where it is used elsewhere: https://github.com/astraw/stdeb/blob/1bd415f852a90ee17469f630fd7ce83037816ea9/stdeb/util.py#L825-L830. Please update the patch to only check for the `if` branch (the presence of `python3-all`).